Yoast/javascript
# Add Polish dictionary stemmer

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Adds a dictionary with ~60 000 Polish words and their base forms to the Polish stemmer, and replaces the word with its base form (if found) before conducting further rule-based stemming.
* [Yoast SEO Premium] Improves keyphrase recognition of common Polish words.

## Relevant technical choices:

* 

## Test instructions

This PR can be tested by following these steps:

* Make sure all of the following forms of Polish words are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: _pies, psy, psa_
* Make sure all of the following forms of Polish words are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: _bezpieczna, niebezpiecznych_
* Make sure all of the following forms of Polish words are being recognised as forms of the same word when one of them is entered in the keyphrase and others in the title/text/meta-description: _napopularniejszych, popularnych_

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-477
